README.md

```diff
@@ -323,31 +124,47 @@ $ tendermint lite
 
 ## 8. Send Your First Transaction
```

Comment:
I guess it'll also make a sense to move this to the dev docs later

Reply:
Sure thing 👍